packages/component-utilities/src/echarts.js

```diff
@@ -424,6 +424,17 @@ export default (node, option) => {
 
 	const chart = init(node, 'evidence-light', { renderer: 'svg' });
 
+	// If the x-axis of a series is numeric, or a date; ensure that it is in order
+	option.series = option.series.map((s) => {
+		if (typeof s.data[0][0] === 'number') {
+			s.data = s.data.sort((a, b) => a[0] - b[0]);
+		} else if (s.data[0][0] instanceof Date) {
+			s.data = s.data.sort((a, b) => a.getTime() - b?.getTime() ?? 0);
```

Comment:
Shouldn't this be `a[0].getTime() - b[0]?.getTime() ?? 0` or similar?

Reply:
Also the `getTime()` isn't strictly necessary because JS implicitly converts Dates to numbers when you try to do arithmetic on them

Reply:
> Shouldn't this be `a[0].getTime() - b[0]?.getTime() ?? 0` or similar?

Yes

```suggestion
			s.data = s.data.sort((a, b) => a[0].getTime() - b[0]?.getTime() ?? 0);
```

> Also the `getTime()` isn't strictly necessary because JS implicitly converts Dates to numbers when you try to do arithmetic on them

True, but this makes it explicit, which I think is helpful when you get into the weeds like this code is